### PR TITLE
general proofreading

### PR DESCRIPTION
**Pull request by Grammaresque**
#### Author checklist (Completed by original Author)
- [x ] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [ x] If applicable, steps and instructions have been tested to work
- [ x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

**Comment by EzequielBruni** [comment]
The grammar changes look great, but we need to fix those URLs. Anyone know where to find the working links? @alemorvan?

File: docs/books/admin_guide/03-commands.md [hunk 1]
```diff
@@ -35,42 +35,42 @@ Current Linux systems have graphical utilities dedicated to the work of an admin
...
-* execute the command;
-* display the command prompt;
-* etc.
+* Display the command prompt.
+* Reading the command.
+* Analyze syntax.
+* Substitute special characters.
+* Execute the command.
+* Display the command prompt.
+* Etc.
```
Comment (sspencerwire) [inline docs/books/admin_guide/03-commands.md:73]: @Grammaresque great work! I'm wondering if the these should be changed to plurals. This is just a suggestion:
Displays the command prompt.
Reads the command.
Analyzes the syntax.
Substitutes special characters.
Executes the command.
Displays the command prompt.

Thoughts?

**Comment by sspencerwire** [comment]
@Grammaresque I just had one question on the document. I've reviewed the whole thing. If you could respond to that one comment, I think it is good to merge.  Thank you very much!

**Comment by sspencerwire** [comment]
> The grammar changes look great, but we need to fix those URLs. Anyone know where to find the working links? @alemorvan?

* The one in the README.md file should already be fixed (https://ersei.saggis.com/). I'll double check to make sure.
* This one should be changed as well. The mirror no-longer contains references to 8.6, but does contain 8.7. I'll double check that one too. It is embedded in this document if you want to take a look and see how it appears on your local copy: books/admin_guide/13-softwares.md
* The last one must have been a temporary error, as it now resolves fine: http://www.gnu.org/software/hello/hello.html

Do you have time to take a look at the other two and just verify?

Thanks Ezequiel!

**Review by sspencerwire — DISMISSED** [review]
Other than that one section that I asked about, this all looks very good. Ready to merge soon.

File: docs/books/admin_guide/03-commands.md (hunk 1 above)
Comment (Grammaresque) [reply docs/books/admin_guide/03-commands.md:73]: Yes! I was trying to make them all consistent in that present tense and I did miss this list. Am I able to fix that or do you do it now that I've already submitted?

**Review by SergeCroise — DISMISSED** [review]

File: docs/books/admin_guide/03-commands.md [hunk 2]
```diff
@@ -35,42 +35,42 @@ Current Linux systems have graphical utilities dedicated to the work of an admin
...
+Once the user is connected to a console, the shell displays the **command prompt**. It then behaves like an infinite loop, repeating the same pattern with each statement entered:
 
-* displaying the command prompt;
-* reading the command;
-* syntax analysis;
-* substitution of special characters;
-* execute the command;
-* display the command prompt;
-* etc.
+* Display the command prompt.
```
Comment (SergeCroise) [inline docs/books/admin_guide/03-commands.md:67]:
```suggestion
* Displays the command prompt.
```

File: docs/books/admin_guide/03-commands.md [hunk 3]
```diff
@@ -35,42 +35,42 @@ Current Linux systems have graphical utilities dedicated to the work of an admin
...
+* Reading the command.
+* Analyze syntax.
+* Substitute special characters.
```
Comment (SergeCroise) [inline docs/books/admin_guide/03-commands.md:70]:
```suggestion
* Substitutes special characters.
```

File: docs/books/admin_guide/03-commands.md [hunk 4]
```diff
@@ -35,42 +35,42 @@ Current Linux systems have graphical utilities dedicated to the work of an admin
...
+* Execute the command.
+* Display the command prompt.
```
Comment (SergeCroise) [inline docs/books/admin_guide/03-commands.md:72]:
```suggestion
* Displays the command prompt.
```

**Review by sspencerwire — APPROVED** [review]
Resolved changes...

**Pull request merged**